scorer/runner.py

```diff
@@ -4,96 +4,52 @@
-                             'raw_rank': group.raw_rank, 'degree': group.degree})
-    db.aql.execute("""
-    FOR u in users
-        FILTER u.score > 90 && ( !u.verifications || 'NodeOne' not in u.verifications || 'BrightID' not in u.verifications ) 
-        update u with { verifications: append(u.verifications, ['NodeOne', 'BrightID'], true) } in users
-        OPTIONS { exclusive: true }
-    """)
-    db.aql.execute("""
-    FOR dfe in users
-        FILTER dfe.dfeAdmin == true
-        FOR c in connections
-            FILTER ( c._from == dfe._id || c._to == dfe._id ) && c.timestamp > 1564600000000
-            FOR u in users
-                FILTER ( u._id == c._from || u._id == c._to )
-                && ( !u.verifications || 'DollarForEveryone' not in u.verifications )
-                update u with { verifications: append(u.verifications, ['DollarForEveryone'], true) } in users
-                OPTIONS { exclusive: true }
-    """)
-    db.aql.execute("""
-    FOR dfe in users
-        FILTER dfe.dfeAdminB == true
-        FOR c in connections
-            FILTER ( c._from == dfe._id || c._to == dfe._id ) && c.timestamp > 1564600000000
-            FOR u in users
-                FILTER ( u._id == c._from || u._id == c._to )
-                && 'BrightID' in u.verifications
-                && 'DollarForEveryone' not in u.verifications
-                update u with { verifications: append(u.verifications, ['DollarForEveryone'], true) } in users
-                OPTIONS { exclusive: true }
-    """)
-
-
-def stupid_sybil_border(graph):
-    border = 0
-    reset_ranks(graph)
-    ranker = algorithms.GroupSybilRank(graph)
-    ranker.rank()
-    attackers = sorted(graph.nodes, key=lambda n: n.rank, reverse=True)
-    for attacker in attackers:
-        attacker.groups.add('stupid_sybil')
-        sybil1 = Node('stupid_sybil_1', 'Sybil', set(['stupid_sybil']))
-        sybil2 = Node('stupid_sybil_2', 'Sybil', set(['stupid_sybil']))
-        graph.add_edge(attacker, sybil1)
-        graph.add_edge(attacker, sybil2)
-        reset_ranks(graph)
-        ranker = algorithms.GroupSybilRank(graph)
-        ranker.rank()
-        border = max(sybil1.raw_rank, sybil2.raw_rank)
-        graph.remove_nodes_from([sybil1, sybil2])
-        attacker.groups.remove('stupid_sybil')
-        reset_ranks(graph)
-        print('attacker: {}\t border: {}'.format(attacker, border))
-        if border:
-            return border
+db = ArangoClient().db('_system')
 
 
 def process(fname):
     json_graph = from_dump(fname)
     graph = from_json(json_graph)
-    border = stupid_sybil_border(graph)
-    raw_ranks = [node.raw_rank for node in graph.nodes]
-    print('''stupid border: {}
-max: {}
-min: {}
-avg: {}'''.format(border, max(raw_ranks), min(raw_ranks), sum(raw_ranks) / len(raw_ranks)))
     reset_ranks(graph)
-    ranker = algorithms.GroupSybilRank(graph, {
-        'stupid_sybil_border': border
-    })
+    ranker = algorithms.Yekta(graph, {})
```

Comment:
Is Yekta with a score of 3 comparable to SybilGroupRank with a score of 10?
https://github.com/BrightID/BrightID-Node/blob/bbe3dceb71b19ddca1208992658e2209c034bd82/scorer/verifications/brightid.py#L12

Reply:
Hmmm
We had a `stupid_sybil_border` in previous implementation that we used to get verification border for SybilRank and we gave score <10 to nodes with ranks lower than the border and >90 to users with ranks higher than border.
In Yekta we defined borders constantly here:
https://github.com/BrightID/BrightID-AntiSybil/blob/ae04b3dff9ddfbbc6816df3564c4d4293668c642/anti_sybil/algorithms/yekta.py#L4

Scores in Yekta are almost number of connections (weakening suspicious ones) so these numbers are basically number of connections so it makes sense to adjust them manually.